### DISCUSSION TITLE
Star Rating Inconsistency

### DISCUSSION THREAD

**Discussion by oku22** [ppy/osu#15891] [Q&A] [answered]
Just raising this discussion to ask about the new star ratings introduced to the game. 

When referring back to the news post about the star rating changes, the star ratings given for the example maps are different from the ones actually shown on the beatmap’s page. For example, Sotarks’ “Immortal Flame” top difficulty on the news post was supposed to update to 6.94*, however on the beatmap page it was updated to 6.60*. 

Would love to hear about what the issue may be, Thanks.

**Comment by smoogipoo** [comment, marked as answer]
The star ratings in the newspost for that section specifically are relating to the old algorithm, i.e. excluding all other changes in the newspost. osu!web is accurate.
